### ISSUE TITLE
breaking changes in v1.0.0 but no documentation

### ISSUE THREAD

**Issue by Progdrasil** [cakecatz/flex-toolbar#137]
the update to flex-tool-bar v1.0.0 seems to remove cson support in favor of js files. However, this breaks any current config depending on a .cson file and there does not seem to be any documentation or examples of how to make a .json or a .js file that works. 

converting my existing .cson file to .js and exporting the existing array with module.exports does not seem to work.

Atom    : 1.24.0
Electron: 1.8.2
Chrome  : 59.0.3071.115
Node    : 8.2.1
tool-bar: 1.1.6
flex-tool-bar: 1.0.0

**Comment by UziTech** [comment]
v1.0.0 still supports cson. Can you share your configuration?

**Comment by Progdrasil** [comment]
here is my configuration
```cson
[
	{
		type: 'button'
		tooltip: 'New File'
		callback: 'tree-view:add-file'
		icon: 'file-code-o'
		iconset: 'fa'
	},
	{
		type: 'button'
		tooltip: 'New Folder'
		callback: 'tree-view:add-folder'
		icon: 'folder'
		iconset: 'fa'
	},
	{
		type: 'button'
		tooltip: 'Add Project Folder'
		callback: 'application:add-project-folder'
		icon: 'folder-open'
		iconset: 'fa'
	},
	{
		type: 'spacer'
	},
	{
		type: 'button'
		icon: 'refresh'
		callback: 'sync-settings:backup'
		tooltip: 'Backup settings'
		iconset: 'fa'
	},
	{
		type: 'button'
		icon: 'cloud-download'
		callback: 'sync-settings:restore'
		tooltip: 'Restore settings'
		iconset: 'fa'
	},
	{
		type: 'spacer'
	},
	{
		type: 'button'
		icon: 'bars'
		callback: 'flex-tool-bar:edit-config-file'
		tooltip: 'Edit Tool Bar'
		iconset: 'fa'
	},
	{
		type: 'button'
		icon: 'cog'
		callback: 'settings-view:open'
		tooltip: 'Open Atom Settings'
		iconset: 'fa'
	},
	{
		type: 'spacer'
	}
]

```

**Comment by UziTech** [comment]
That configuration in a cson file seems to work for me on v1.0.0

![image](https://user-images.githubusercontent.com/97994/37161932-9b37cb50-22b9-11e8-85f1-0a39e2a9baae.png)

Were you getting an error?

**Comment by Progdrasil** [comment]
i get this error the second i update to 1.0.0

### Stack Trace

Failed to activate the tool-bar package

```
At snapshotResult is not defined

ReferenceError: snapshotResult is not defined
    at Object.removeCache (/packages/flex-tool-bar/lib/flex-tool-bar.js:540:3)
    at Object.loadConfig (/packages/flex-tool-bar/lib/flex-tool-bar.js:564:9)
    at Object.reloadToolbar (/packages/flex-tool-bar/lib/flex-tool-bar.js:426:32)
    at Object.consumeToolBar (/packages/flex-tool-bar/lib/flex-tool-bar.js:410:15)
    at Provider.module.exports.Provider.provide (/usr/lib/atom/node_modules/service-hub/lib/provider.js:34:52)
    at ServiceHub.module.exports.ServiceHub.provide (/usr/lib/atom/node_modules/service-hub/lib/service-hub.js:30:20)
    at Package.activateServices (/usr/lib/atom/src/package.js:392:69)
    at Package.activateNow (/usr/lib/atom/src/package.js:228:14)
    at measure (/usr/lib/atom/src/package.js:201:27)
    at Package.measure (/usr/lib/atom/src/package.js:83:19)
    at activationPromise.Promise (/usr/lib/atom/src/package.js:195:14)
    at Promise (<anonymous>)
    at Package.activate (/usr/lib/atom/src/package.js:193:32)
    at PackageManager.activatePackage (/usr/lib/atom/src/package-manager.js:684:36)
    at config.transactAsync (/usr/lib/atom/src/package-manager.js:659:30)
    at Config.module.exports.Config.transactAsync (/usr/lib/atom/src/config.js:365:18)
    at PackageManager.activatePackages (/usr/lib/atom/src/package-manager.js:657:17)
    at PackageManager.activate (/usr/lib/atom/src/package-manager.js:636:44)
    at loadStatePromise.loadState.then (/usr/lib/atom/src/atom-environment.js:827:21)
    at <anonymous>
```
### Non-Core Packages

```
atom-ide-ui 0.9.3 
atom-material-ui 2.1.3 
atom-typescript 12.3.5 
file-icons 2.1.17 
flex-tool-bar 1.0.0 
highlight-selected 0.13.1 
indent-guide-improved 1.4.13 
intentions 1.1.5 
language-babel 2.84.0 
language-docker 1.1.8 
language-ejs 0.4.0 
language-matlab 0.2.1 
language-rust 0.4.12 
language-swift 0.5.0 
language-vue 0.23.1 
linter 2.2.0 
linter-eslint 8.4.1 
linter-ui-default 1.7.1 
minimap 4.29.8 
minimap-highlight-selected 4.6.1 
monokai 0.24.0 
monokai-ide-syntax 1.0.1 
pigments 0.40.2 
sync-settings 0.8.5 
todo-show 2.2.0 
tool-bar 1.1.6 
```

**Comment by UziTech** [comment]
I think this is fixed in v1.1.0

It looks like that is a bug from converting our code to js from coffeescript

Out of curiosity are you using Atom in dev mode? I'm trying to figure out why snapshotResult wouldn't be defined by Atom.

**Comment by Progdrasil** [comment]
i never used atom in dev mode, i dont know what could have caused that
